Representable/representable
# fix hover bug

fixes #352 

**to test**
- go to entry page
- search in one state
- verify hover works
- search in other state
- verify hover still working
- go to /entry/MI/
- verify hover works
